### DISCUSSION TITLE
Feature Request - multiple emails with tag +something

### DISCUSSION THREAD

**Discussion by impedator** [osTicket/osTicket#6286] [Ideas]
Many of email servers/providers allow automatic tagging system with "+" sign - some.address+tag@somedomain.com.
It will be great if osTicket allow to create multiple e-mails (example - some.address+tag1@somedomain.com some.address+tag2@somedomain.com) based on one e-mail account (some.address@somedomain.com)

**Comment by JediKev** [comment]
@impedator 

With system emails you can already do so however if you configure the login info for basic auth the username has to be unique. Also, for now, with OAuth2 the system email must be a real, unique email as we use that to confirm you are authorizing the correct account. I would also like to mention that you cannot login/authenticate with `+` emails so you wouldn't be able to configure IMAP/SMTP anyways. This is not an application restriction, rather a restriction on the provider's end.

With user/agent accounts this is possible as well.

Cheers.

**Reply by impedator** [reply]
@JediKev 
Thanks for quick answer, but I think that I have misguide you with my initial post. Lets see an example:

Company A has an e-mail address for customer support - support@companyA.com
imap and smtp i authenticated with this address support@companyA.com and superSecretPassword123! password.
on mail server are configured IMAP folders productA, producB and productC, and automatic sieve filter that redirecting/storing emails to this folders support+productA@companyA.com support+productB@companyA.com support+productC@companyA.com

My idea is to configure in osTicket three emails, all authenticated with the same credentials (same server, same account), but fetching messages from different folders and assigning tickets to different teams based on those folders.

Best
i
